k8s/service_controller/Makefile

```diff
@@ -24,12 +24,25 @@ test-unit: binary
 	  -v ${CURDIR}:/src/github.com/cncf/servicebroker/k8s/service_controller \
 	  -e GOPATH=/ \
 	  golang \
-	  sh -c "go get ./... && make --no-print-directory test-unit-local"
+	  sh -c "go get -d ./... && make --no-print-directory test-unit-local"
 	
 test-unit-local:
-	@TEST_DIRS=$(shell find . -name \*_test.go -exec dirname {} \; | sort -u)
+	@TEST_UNIT_DIRS="$(shell find . -name \*_test.go -exec dirname {} \; | sort -u)"
 	@echo "\n### Running the unit tests"
-	@go test $(TEST_DIRS)
+	@go test $(TEST_UNIT_DIRS)
+
+test: binary
```

Comment:
let's deal with that in a separate PR - I want to get some of these re-org changes in.